### PR TITLE
TESHARIIIIII

### PR DESCRIPTION
**Pull request by Tokorizo**
## About The Pull Request

ill update this later, wip for Tesh species

## Why It's Good For The Game

read above

## Changelog

:cl:
add: Added new mechanics or gameplay changes
add: Added more things
expansion: Expands content of an existing feature
del: Removed old things
qol: made something easier to use
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

File: code/_globalvars/lists/flavor_misc.dm [hunk 1]
```diff
@@ -42,6 +42,11 @@ GLOBAL_LIST_EMPTY(caps_list)
 GLOBAL_LIST_EMPTY(tails_list_monkey)
 GLOBAL_LIST_EMPTY(pod_hair_list)
 
+//PARIAH EDIT ADDITION
+GLOBAL_LIST_EMPTY(teshari_feathers_list)
+GLOBAL_LIST_EMPTY(teshari_tail_list)
```
Comment (SabreML) [inline code/_globalvars/lists/flavor_misc.dm:47]: Reminder to implement `teshari_tail_list` at some point

File: modular_pariah/master_files/code/modules/mob/living/carbon/human/species_type/teshari.dm [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		BODY_ZONE_R_ARM = /obj/item/bodypart/r_arm/teshari,
+		BODY_ZONE_L_LEG = /obj/item/bodypart/l_leg/teshari,
+		BODY_ZONE_R_LEG = /obj/item/bodypart/r_leg/teshari,
+	)
+
+/datum/species/teshari/get_species_description()
+	return "The militaristic Lizardpeople hail originally from Tizira, but have grown \
+		throughout their centuries in the stars to possess a large spacefaring \
+		empire: though now they must contend with their younger, more \
+		technologically advanced Human neighbours."
```
[the remark this answers is not in the document]
Comment (Tokorizo) [reply modular_pariah/master_files/code/modules/mob/living/carbon/human/species_type/teshari.dm:40]: Just the base, we will edit it later.

File: modular_pariah/master_files/code/modules/mob/living/carbon/human/species_type/teshari.dm [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+		"The history between lizards and humans has resulted in many conflicts that lizards ended on the losing side of, \
+		with the finale being an explosive remodeling of their moon. Today's lizard-human relations are seeing the continuance of a record period of peace.",
+
+		"Lizard culture is inherently militaristic, though the influence the military has on lizard culture \
+		begins to lessen the further colonies lie from their homeworld - \
+		with some distanced colonies finding themselves subsumed by the cultural practices of other species nearby.",
+
+		"On their homeworld, lizards celebrate their 16th birthday by enrolling in a mandatory 5 year military tour of duty. \
+		Roles range from combat to civil service and everything in between. As the old slogan goes: \"Your place will be found!\"",
+	)
```
[the remark this answers is not in the document]
Comment (Tokorizo) [reply modular_pariah/master_files/code/modules/mob/living/carbon/human/species_type/teshari.dm:59]: see above

File: code/modules/mob/living/carbon/human/human_update_icons.dm [hunk 4]
```diff
@@ -293,7 +293,7 @@ There are several things that need to be remembered:
 		var/handled_by_bodytype = TRUE
 		var/icon_file
 
-		if(dna.species.bodytype & BODYTYPE_TESHARI)
+		if(dna.species.bodytype & BODYTYPE_CUSTOM)
```
Comment (Kapu1178) [inline code/modules/mob/living/carbon/human/human_update_icons.dm:296]: why

Comment (Tokorizo) [reply code/modules/mob/living/carbon/human/human_update_icons.dm:296]: was taking reference from a SR PR that handles both Vox and Teshari, so it looked like it was a simpler option

Comment (Kapu1178) [reply code/modules/mob/living/carbon/human/human_update_icons.dm:296]: This is objectively not how my limb system works. This will break shit.

Comment (Tokorizo) [reply code/modules/mob/living/carbon/human/human_update_icons.dm:296]: gotcha, ill revert it back to what it was

**Comment by SabreML** [comment]
Let me know when this is ready btw and I'll do a full review on it.

**Comment by Tokorizo** [comment]
Fuck my life

**Comment by Tokorizo** [comment]
Sabre is doing it now in #688

**Pull request closed without merging**